java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -602,6 +605,199 @@ public static CompletableFuture<Object> arrinsert(
                         .toArray());
     }
 
+    /**
+     * Searches for the first occurrence of a <code>scalar</code> JSON value in the arrays at the
+     * path.
+     *
+     * @param client The client to execute the command.
+     * @param key The key of the JSON document.
+     * @param path The path within the JSON document.
+     * @param scalar The scalar value to search for; JSON scalar refers to values that are not objects
+     *     or arrays.
+     * @return An array of integers. Each value is the index of the matching element in the array at
+     *     the path. The value is -1 if not found. If there are no values in the array, the return
+     *     value will be <code>null</code>
+     * @example
+     *     <pre>{@code
+     * Json.set(client, "doc", "$", "{\"a\": [\"value\", 3], \"b\": {\"a\": [3, [\"value\", false], 5]}}").get();
+     * Long[] result = Json.arrindex(client, key, "$..a", new JsonScalar(3)).get();
+     * assertArrayEquals(new Long[] {Long.valueOf(1), Long.valueOf(0)}, result);
+     * }</pre>
+     */
+    public static CompletableFuture<Long[]> arrindex(
```

Comment:
```suggestion
    public static CompletableFuture<Object> arrindex(
```
It can return `Long[]` or `Long`

Reply:
i feel like the return should be Object[]